holoviews/plotting/bokeh/element.py

```diff
@@ -1634,17 +1634,20 @@ class ColorbarPlot(ElementPlot):
         Number of discrete colors to use when colormapping or a set of color
         intervals defining the range of values to map each color to.""")
 
+    cformatter = param.ClassSelector(
+        default=None, class_=(util.basestring, TickFormatter, FunctionType), doc="""
+        Formatter for ticks along the colorbar axis.""")
+
     clabel = param.String(default=None, doc="""
         An explicit override of the color bar label, if set takes precedence
         over the title key in colorbar_opts.""")
 
     clim = param.Tuple(default=(np.nan, np.nan), length=2, doc="""
-       User-specified colorbar axis range limits for the plot, as a tuple (low,high).
-       If specified, takes precedence over data and dimension ranges.""")
+        User-specified colorbar axis range limits for the plot, as a tuple (low,high).
+        If specified, takes precedence over data and dimension ranges.""")
 
-    cformatter = param.ClassSelector(
-        default=None, class_=(util.basestring, TickFormatter, FunctionType), doc="""
-        Formatter for ticks along the colorbar axis.""")
+    cnorm = param.ObjectSelector(default='linear', objects=['linear', 'log', 'eqhist'], doc="""
```

Comment:
I half agree...it is an annoying extra character!

Reply:
Alternatively put, is it worth making the specification longer for HoloViews and hvPlot (the way we want to do things going forward) due to the history of the thing we are moving away from (server side eqhist within datashader)?

Reply:
Pros for `eq_hist`:
- Matches datashader, so that in e.g. hvPlot and HoloViews docs we can say `eq_hist` consistently to refer to the equivalent operation whether it's done by `shade` or by Bokeh.
- Gives a hint to pronunciation and meaning. In a list with "linear" and "log", the default pronuciation of "eqhist" would be something like "eckhist", which would just be weird. `eq_hist` makes it clearer that it's an abbreviation of two words, and then people can guess or at least more easily remember what those two words are.

Pros for `eqhist`:
- One character shorter.

Seems like a clear choice to me :-), but I'll let @philippjfr cast the tiebreaking vote.

Reply:
+1 for `eq_hist`

Reply:
Last suggestion before I give up and use `eq_hist`: why not just `eq`? Where it is `eq_hist` or `eq`, a newcomer needs to look up what it means regardless.

That would mean 'equalized' where the fact it is 'histogram equalization' isn't particularly important (which would also suggest the name should be `hist_eq`!

Reply:
Such a struggle over a few characters! 🙂 'eq' seems unsearchable, so I don't prefer that, plus it doesn't match datashader.

Reply:
I renamed 'eqhist' to 'eq_hist' in 204ac6c. I don't like that string much but I'm happy for this to be merged as long as you have considered possible alternatives.